getting-started/main.cpp

```diff
@@ -142,9 +142,10 @@ static void sign_a_message_using_rsa(void)
 {
     psa_status_t status;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
-    uint8_t key[] = RSA_KEY;
+    static const uint8_t key[] = RSA_KEY;
     uint8_t hash[] = "INPUT_FOR_SIGN";
-    uint8_t signature[PSA_ASYMMETRIC_SIGNATURE_MAX_SIZE] = {0};
+    /* 'signature' is static to reduce stack usage. */
+    static uint8_t signature[PSA_ASYMMETRIC_SIGNATURE_MAX_SIZE] = {0};
```

Comment:
Making `signature` static is not necessary for the targets we run this example on so far. So, if we dislike adding static here, we can remove it as we don't need it yet. Just making the key `static const` is good enough to pass the tests with ARMC6-develop at least.